Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Correcting path to lxc-init in the sshd template (Closes: #697267).

diff -Naurp lxc.orig/templates/lxc-sshd.in lxc/templates/lxc-sshd.in
--- lxc.orig/templates/lxc-sshd.in
+++ lxc/templates/lxc-sshd.in
@@ -170,7 +170,7 @@ fi
 
 if [ $0 == "/sbin/init" ]; then
 
-    type @LXCINITDIR@/lxc-init
+    type @LXCINITDIR@/lxc/lxc-init
     if [ $? -ne 0 ]; then
         echo "'lxc-init is not accessible on the system"
         exit 1
@@ -198,7 +198,7 @@ EOF
         dhclient eth0 -cf /dhclient.conf
     fi
 
-    exec @LXCINITDIR@/lxc-init -- /usr/sbin/sshd
+    exec @LXCINITDIR@/lxc/lxc-init -- /usr/sbin/sshd
     exit 1
 fi
 
